debian/rules

```diff
@@ -7,3 +7,6 @@
 
 override_dh_auto_test:
 	./runtests pep8 small
+
+override_dh_installman:
+	help2man -o debian/ubuntu-make.1 umake
```

Comment:
I will readd a dh_installman call under this.
```
override_dh_installman:
	help2man -o debian/ubuntu-make.1 umake
        dh_installman
```

That way, the default action is still present.